packages/sign-client/src/constants/engine.ts

```diff
@@ -165,4 +167,70 @@ export const TVF_METHODS = {
   solana_signAndSendTransaction: {
     key: "signature",
   },
+
+  // sui
+  sui_signAndExecuteTransaction: {
+    key: "digest",
+  },
+  sui_signTransaction: {
+    key: "",
+  },
+
+  // hedera
+  hedera_signAndExecuteTransaction: {
+    key: "transactionId",
+  },
+  hedera_executeTransaction: {
+    key: "transactionId",
+  },
+
+  // near
+  near_signTransaction: {
+    key: "",
+  },
+  near_signAndExecuteTransaction: {
+    key: "",
+  },
+  near_signAndExecuteTransactions: {
+    key: "",
+  },
```

Comment:
for Near the RPC methods are: `near_signTransaction` and `near_signTransactions`
and the response payload includes: `type` and `data` fields

https://www.notion.so/walletconnect/Near-TVF-1ee3a661771e8043a5fad110554e128a?source=copy_link